awx/ui_next/src/components/ErrorDetail/ErrorDetail.jsx

```diff
@@ -84,7 +88,10 @@ function ErrorDetail({ error }) {
 }
 
 ErrorDetail.propTypes = {
-  error: PropTypes.instanceOf(Error).isRequired,
+  error: PropTypes.instanceOf(Error),
+};
+ErrorDetail.defaultProps = {
+  error: null,
```

Comment:
With the change to `fail tests if warning is logged` could we keep this a required prop?

Reply:
Oh nice, I hadn't seen that PR. Though I figured since we obviously need that conditional check everywhere, we might as well do it once inside the component instead of all over the place